libpod/container_config.go

```diff
@@ -153,6 +153,8 @@ type ContainerRootFSConfig struct {
 	Secrets []*ContainerSecret `json:"secrets,omitempty"`
 	// SecretPath is the secrets location in storage
 	SecretsPath string `json:"secretsPath"`
+	// StorageOpts to be used when creating rootfs
+	StorageOpts []string `json:"storageOpts"`
```

Comment:
I think this should be a `map[string]string` given the c/storage accepts this, to avoid converting the array every time.